c10/core/TensorImpl.h

```diff
@@ -408,6 +408,10 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
     return numel_;
   }
 
+  virtual int64_t element_size() const {
```

Comment:
Why do we need to add this function? (Does TorchScript `Tensor.element_size()` directly bind to functions in TensorImpl?) We have `Tensor::element_size()` defined in `aten/src/ATen/templates/TensorBody.h`, is there a way to use that instead?

Reply:
@yf225 

The issue describes this missing parity, #33620.
Digging where the scripting error comes from, I noticed it comes from 

https://github.com/pytorch/pytorch/blob/ee28831341c0398b0e2a0791ffd77263217f0bff/torch/csrc/jit/script/sugared_value.cpp#L151

While, the similar `Tensor().numel()` works by running into this line. 
https://github.com/pytorch/pytorch/blob/ee28831341c0398b0e2a0791ffd77263217f0bff/torch/csrc/jit/script/sugared_value.cpp#L145

So I search for where `aten::numel` is registered and noticed that `tensor_impl_methods` list I am touching. And it says it works by adding methods in `TensorImpl` class.,